api/doc/thor.yaml

```diff
@@ -736,6 +739,36 @@ paths:
                 type: string
                 example: 'Invalid address'
 
+
+  /metrics:
+    get:
+      tags:
+        - Telemetry
+      summary: Retrieve Prometheus metrics
+      description: |
+        This endpoint allows clients to retrieve Prometheus metrics.
+      responses:
+        '200':
```

Comment:
We could describe the error code if telemetry is disabled

Reply:
Yep, I'll add that in the parent PR 👍